ts/components/select/index.ts

```diff
@@ -294,7 +295,8 @@ export default (initOptions: InitOptions): Select => ({
 
     const request = new Request(url, {
       method,
-      body: method === 'POST' ? JSON.stringify(parameters) : undefined
+      body: method === 'POST' ? JSON.stringify(parameters) : undefined,
+      credentials: credentials ?? 'omit'
```

Comment:
Just add the credentials value, keep undefined as default
```suggestion
      credentials
```